Source/ASControlNode.mm

```diff
@@ -316,7 +329,6 @@ - (void)addTarget:(id)target action:(SEL)action forControlEvents:(ASControlNodeE
       }
     });
 
-  self.userInteractionEnabled = YES;
```

Comment:
This line has a very controversial history. Let's leave it as-is and deal with it in another PR if need be.

https://github.com/facebook/AsyncDisplayKit/pull/1189